container/features/src/main/resources/features.xml

```diff
@@ -551,6 +551,15 @@
       <bundle dependency="true">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.kafka-clients/0.10.1.1_1</bundle>
     </feature>
 
+    <feature name='opennms-aws-sqs' description="OpenNMS :: Features :: AWS SQS" version="${project.version}">
+      <bundle dependency="true">wrap:mvn:com.amazonaws/aws-java-sdk-core/1.11.209$Bundle-Version=2.2.0&amp;Export-Package=*;-noimport:=true;version="1.11.209"</bundle>
```

Comment:
Can we make the aws-java-sdk version a property in the root pom.xml instead of referencing the specific version here?

Reply:
Absolutely! Interestingly, the Kafka version doesn't have it, which is what I've used as a reference. To be consistent, someone should fix Kafka, I think.

Reply:
I would strip off all of the metadata after (and including) the `$`, I don't think that any of that should normally be necessary. I see that you were copying the kafka feature but I'm not sure why it's there either. Do that for every bundle in this feature.